runtime/runtimetools/perfscripts/perf-hottest

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+
+# ------------------------------------------------------------------------
+ # Copyright (c) 2020, 2020 IBM Corp. and others
+ #
+ # This program and the accompanying materials are made available under
+ # the terms of the Eclipse Public License 2.0 which accompanies this
+ # distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ # or the Apache License, Version 2.0 which accompanies this distribution and
+ # is available at https://www.apache.org/licenses/LICENSE-2.0.
+ #
+ # This Source Code may also be made available under the following
+ # Secondary Licenses when the conditions for such availability set
+ # forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ # General Public License, version 2 with the GNU Classpath
+ # Exception [1] and GNU General Public License, version 2 with the
+ # OpenJDK Assembly Exception [2].
+ #
+ # [1] https://www.gnu.org/software/classpath/license.html
+ # [2] http://openjdk.java.net/legal/assembly-exception.html
+ #
+ # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+# ---------------------------------------------------------------------------
+
+# This script checks to make sure the following conditions:
```

Comment:
Code looks fine - could you just add some comment documentation on what this tool does and what the different modes are? With that I think this should be good to go.

Reply:
Added the comments after the legal paragraph.  @andrewcraik

Reply:
@BeverlyXu could you please wrap the inlines at some convenient width so it can be read without scrolling - 120 chars is common, but just to make this easier to read.